OKR.md

```diff
@@ -0,0 +1,50 @@
+# Quarterly Objectives and Key Results
+
+We try to frame our ongoing work using a process based on quarterly Objectives and Key Results (OKRs). Objectives reflect outcomes that are challenging, but realistic. Results are tangible and measurable.
+
+## 2018 Q2
+
+> Work in Progress
+
+### The daemon is stable and so core is reliable
+
+- `PX` - OWNER - The uncaught errors and memory usage are under control and a daemon can run for a week non stop
+- `PX` - OWNER - One or more js-ipfs daemons are part of the IPFS infrastructure
+- `PX` - OWNER - js.ipfs.io exposes an IPFS Gateway provided by a js-ipfs daemon
```

Comment:
@VictorBjelkholm can I assign you on this one given that you already started this work with #804?

Reply:
:+1: Should we have one bootstrapper node be a js-ipfs node as well? My thinking is that the js-ipfs gateway probably won't be used a lot, while a default bootstrap node would get a lot of usage

Reply:
See KR on L12 `One or more js-ipfs daemons are part of the IPFS infrastructure`. That should be it :)

Reply:
Ah, I would make it more explicit, include "bootstrap" somewhere in the text :+1: